it/test/repositories/SessionDataRepositoryISpec.scala

```diff
@@ -50,87 +50,65 @@ class SessionDataRepositoryISpec extends AnyWordSpec
 
   val testAlternativeSessionId = "session-xxx"
 
-  override val testSession: Session = Session(
-    sessionId = testSessionId,
-    mtditid = "testId",
-    nino = "testNino",
-    utr = "testUtr",
-    internalId = "test-internal-id"
-  )
+  val crypter = SymmetricCryptoFactory.aesGcmCrypto("QmFyMTIzNDVCYXIxMjM0NQ==")
```

Comment:
if we can move this encryption key to the variable, I understand this is a spec and we can do "short cuts" but worth to keep these separately